docs/UserGuide.md

```diff
@@ -185,89 +188,93 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+You can add a person to your address book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-
-<box type="tip" seamless>
-**Tip:** A person can have any number of tags (including 0)
-</box>
-
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+<box type="tip" seamless>
+**Tip:** You can add any number of tags to a person! (including 0)
+</box>
+
+### Listing All Persons : `list`
 
-Shows a list of all persons in the address book.
+You can list out all persons in your address book.
```

Comment:
perhaps instead of writing "person", we can write "student"? that would better fit our use cases, also for this particular line there seems to be a grammatical mistake